Readme.md

```diff
@@ -31,10 +31,14 @@ Here's an example of what an app shows up as on the Plex web interface
 
 The rows in that example from top to bottom are `deviceName`, `version`, `product`, and `device`.
 
-### .query(uri)
+### .query(parameters)
```

Comment:
I only went with parameters because that's what WebStorm did when I used its handy "convert to parameters object" feature and then didn't give it a lot more thought - I agree that especially in the context of HTTP stuff, it's not the best name. I'll change it to options!